handlers.go

```diff
@@ -47,6 +47,17 @@ type signatureresponse struct {
 func (a *autographer) handleSignature(w http.ResponseWriter, r *http.Request) {
 	rid := getRequestID(r)
 	starttime := getRequestStartTime(r)
+	auth, userid, authorized, err := a.authorizeHeader(r)
+	if err != nil || !authorized {
+		if a.stats != nil {
+			sendStatsErr := a.stats.Timing("hawk.authorize_header_failed", time.Since(getRequestStartTime(r)), nil, 1.0)
```

Comment:
nit: this can be relative to `starttime` which is already `getRequestStartTime(r)`